### ISSUE TITLE
Can't seem to get the uImage to actually be built with sunxi-3.4 branch

### ISSUE THREAD

**Issue by m34nbunny** [linux-sunxi/linux-sunxi#220]
Hey guys,

I am trying to build the sunxi-3.4 kernel and am running into some issues. Right now I am at the step of building the kernel. I have cloned the source, performed a checkout on sunxi-3.4, ran make mrproper (to ensure I had cleared everything out). Then ran the following command below.

make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- sun4i_defconfig
make -j8 ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- uImage

The build runs and finishes, most articles I have read say that the uImage should be in arch/arm/boot path but there is nothing there. If I compile for the modules, I do end up getting a created output/modules/X.X folder with the appropriate modules built. I pasted the build output below. Any help is greatly greatly appreciated, trying to repurpose my board for something a little more exciting! I hope I have provided enough information.

My current OS is Ubuntu 14.10, running Gnome 3 (if that matters)

Thanks guys/gals!

Following the following documentation. 
http://linux-sunxi.org/Linux_Kernel#Compilation

Below here is the compile output

  LD      arch/arm/plat-sunxi/pm/standby/standby.elf
/media/meanbunny/Storage/source/Meanbunny/linux-sunxi/arch/arm/plat-sunxi/pm/standby/Makefile:22: recipe for target 'arch/arm/plat-sunxi/pm/standby/standby.elf' failed
/media/meanbunny/Storage/source/Meanbunny/linux-sunxi/arch/arm/plat-sunxi/pm/Makefile:8: recipe for target 'arch/arm/plat-sunxi/pm/standby/standby.bin' failed
scripts/Makefile.build:441: recipe for target 'arch/arm/plat-sunxi/pm' failed
Makefile:945: recipe for target 'arch/arm/plat-sunxi' failed

EDITED : Just wanted to clean this post up a bit

**Comment by ryba84** [comment]
Run compilation without -j8. You'll see where is the problem.

**Comment by m34nbunny** [comment]
Testing this now, already have some good progress. Will report back when the build finishes with what I did.

Thanks!!

**Comment by m34nbunny** [comment]
Gotcha, after removing the -j8 I was able to see the issue was related to Standby Power not being able to compile. I changed the config with : make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- menuconfig 

Then went to, Power management options --> and disabled all of them (tried just the first option hoping only standby would be disabled but no luck). After I did this, I got a little further, but now am hitting the following error. 

net/built-in.o: In function `wiphy_rfkill_start_polling':
/media/meanbunny/Storage/source/Meanbunny/linux-sunxi/net/wireless/core.c:637: undefined reference to`rfkill_resume_polling'
Makefile:873: recipe for target '.tmp_vmlinux1' failed
make: **\* [.tmp_vmlinux1] Error 1
